runtime-modules/storage/src/tests/mock.rs

```diff
@@ -130,6 +131,15 @@ impl pallet_timestamp::Trait for Test {
     type MinimumPeriod = MinimumPeriod;
 }
 
+impl common::Trait for Test {
```

Comment:
The content directory and the storage system share these same types so it is a nice idea to have a Trait combining these shared types, but I think we need a better name for the trait type itself.

Reply:
https://github.com/Joystream/joystream/pull/2103/commits/a59562e0991801a84f1a16d91f6326129cca744c